packages/oauth-client/src/CognitoClient.ts

```diff
@@ -46,29 +46,37 @@ export class CognitoClient extends OAuthClient {
   };
 
   /**
-   * Query the Cognito API for the user object associated with an access token.
+   * Query the API for the user object associated with an access token.
    *
-   * @param {string} accessToken - The Cognito access token for Authorization header
+   * @param {Object} params
+   * @param {string} params.token - The access token for Authorization header
+   * @param {string} [params.xRequestId] - a string to help identify the request
    * @returns {Promise<Object>} The user object (see example)
    *
    * @example
    *
    * {
-   *  "username": "Jane Doe",
+   *  "username": "janedoe",
    *  "given_name": "Jane",
    *  "family_name": "Doe",
    *  "study_area": "Atmospheric Composition",
    *  "organization": "NASA",
    *  "email": "janedoe@example.com"
    * }
    */
-  async getUserInfo(accessToken: string) {
-    if (!accessToken) throw new TypeError('accessToken is required');
+  async getUserInfo(params: {
+    token: string,
+    xRequestId?: string,
+  }) {
```

Comment:
Nit: Consider explicit return typing here

Reply:
Not sure if the return values from cognito will be changed

Reply:
Sure, but by typing it we enforce that the documentation and client code explicitly use a type def -- if the client changes that necessitates an update that will rely on that change.  

Apologies if that comment is obtuse, I recognize I might be missing something.